src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -1,15 +1,19 @@
 package name.valery1707.kaitai;
 
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.apache.commons.io.output.TeeOutputStream;
 import org.buildobjects.process.ExternalProcessFailureException;
 import org.buildobjects.process.ProcBuilder;
 import org.buildobjects.process.StartupException;
 import org.buildobjects.process.TimeoutException;
 import org.slf4j.Logger;
 
 import java.nio.file.Path;
+import java.util.Arrays;
 import java.util.LinkedHashSet;
 import java.util.Set;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'java\.nio\.charset\.StandardCharsets\.UTF\_8' is in the wrong order\. Should be in the 'STATIC' group, expecting not assigned imports on this line\.